doc/sphinx/addendum/program.rst

```diff
@@ -194,14 +194,14 @@ Program Fixpoint
    The optional order annotation follows the grammar:
 
    .. productionlist:: orderannot
-      order      : measure `term` (`term`)? | wf `term` `term`
+      order      : measure `term` (`term`)? | wf `term` `ident`
 
-   + :g:`measure f ( R )` where :g:`f` is a value of type :g:`X` computed on
-     any subset of the arguments and the optional (parenthesised) term
-     ``(R)`` is a relation on ``X``. By default ``X`` defaults to ``nat`` and ``R``
+   + :g:`measure f R` where :g:`f` is a value of type :g:`X` computed on
+     any subset of the arguments and the optional term
+     ``R`` is a relation on ``X``. By default ``X`` defaults to ``nat`` and ``R``
```

Comment:
Not sure why it stops using `:g:` at this line :confused:

Reply:
Fixed.